compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -121,16 +111,27 @@ function finishIdentifierStmt(Tokenizer tok, string identifier, Position pos) re
     }
     else if cur == ":" {
         check tok.advance();
-        string name = check tok.expectIdentifier();
-        check tok.expect("(");
-        FunctionCallExpr stmt = check finishFunctionCallExpr(tok, identifier, name, pos);
-        check tok.expect(";");
-        return stmt;
+        return finishQualIdentifierStmt(tok, identifier, check tok.expectIdentifier(), pos);
     }
     else if cur is [IDENTIFIER, string] {
         TypeDescRef ref = { ref: identifier, pos };
         return finishVarDeclStmt(tok, ref);
     }
+    return finishQualIdentifierStmt(tok, (), identifier, pos);
+}
+
+function finishQualIdentifierStmt(Tokenizer tok, string? prefix, string identifier, Position pos) returns Stmt|err:Syntax {
```

Comment:
This is a neat solution. But I would call it finishOptQualIdentifierStmt since it is called for identifiers that are not qualified.